src/test/javascript/spec/component/programming-exercise/programming-exercise-configure-grading.component.spec.ts

```diff
@@ -201,23 +206,23 @@ describe('ProgrammingExerciseConfigureGradingComponent', () => {
         return getElement(debugElement, testCasesNoUpdated);
     };
 
-    beforeEach(async(() => {
+    beforeEach(() => {
         TestBed.configureTestingModule({
-            imports: [TranslateModule.forRoot(), ArtemisTestModule, NgxDatatableModule, NgbModule],
+            imports: [TranslateTestingModule, ArtemisTestModule, NgxDatatableModule, NgbModule],
```

Comment:
If possible, mock the `TranslateTestingModule`. Also have a look whether the `NgbModule` is actually needed. Might be possible to mock some parts of it. I think the `AlertService` can be mocked as well.

Reply:
Thanks, all three of those suggestions were possible actually. I implemented them in 83c02e2dd0eb841bf88d23cbc222584d4e1e94f0.